sway/layout.c

```diff
@@ -279,19 +283,53 @@ void move_container(swayc_t *container, enum movement_direction dir) {
 		sway_log(L_DEBUG, "container:%p, parent:%p, child %p,",
 				container,parent,child);
 		if (parent->layout == layout
+			|| layout == L_NONE /* accept any layout for next/prev direction */
```

Comment:
This won't accept any layout, right? It'll accept no layout, which isn't a thing that happens in practice (things like views have no layout)